doc/nrf/app_dev/device_guides/nrf54l/building_nrf54l.rst

```diff
@@ -0,0 +1,136 @@
+.. _building_nrf54l:
+
+Building and programming with nRF54L15 DK
+#########################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This guide provides instructions on how to build and program the nRF54L15 development kit.
+Whether you are working with single or multi-image builds, the following sections will guide you through the necessary steps.
+
+Depending on the sample, you must program only the application core or both the Fast Lightweight Peripheral Processor (FLPR) and the application core.
+Additionally, the process will differ based on whether you are working with a single-image or multi-image build.
+
+.. note::
+   The following instructions do not include multi-image single-core builds.
+
+This process complements the standard building procedure.
+While building for the application core follows the default process, if you wish to utilize the FLPR core you must use additional configuration to enable it.
+
+Building for the application core
+*********************************
+
+For instructions on building for the application core only, see how to :ref:`build an application <building>`.
+
+Building for the application and FLPR core
+******************************************
+
+This section outlines how to build and program for both the application and FLPR core, covering separate builds and sysbuild configurations.
+FLPR core supports two variants:
+
+* ``nrf54l15dk/nrf54l15/cpuflpr`` where FLPR runs from SRAM, which is the recommended method.
+  To build FLPR image with this variant, the application core image must include the ``nordic-flpr`` snippet.
+
+* ``nrf54l15dk/nrf54l15/cpuflpr/xip`` where FLPR runs from RRAM.
+  To build FLPR image with this variant, the application core image must include the ``nordic-flpr-xip`` snippet.
+
+Standard build
+--------------
+
+This subsection focuses on how to build an application using sysbuild.
+
+.. note::
+   We recommend using FLPR as the main CPU in multi-core projects.
+   Currently, the documentation does not cover specific instructions for building an application image that uses sysbuild to incorporate the FLPR core as a sub-image.
```

Comment:
I wouldn't say we recommend it. I would stick to the 2nd sentence only to describe the limitation of the current solution.
```suggestion
   Currently, the documentation does not cover specific instructions for building an application image that uses sysbuild to incorporate the FLPR core as a sub-image.
   The only documented scenario is for building FLPR as the main CPU and the application as a sub-image.
```

Reply:
I would rephrase last sentence to `FLPR as the main image and the application as a sub-image`.
FLPR is supposed to be a coprocessor, so calling it main CPU is misleading.
APP can be a sub-image and still do most of the work.

Reply:
applied as suggested